python/ray/util/sgd/torch/torch_trainer.py

```diff
@@ -150,7 +146,6 @@ def __init__(
             timeout_s=NCCL_TIMEOUT_S,
             use_fp16=False,
             use_tqdm=False,
-            apex_args=None,
```

Comment:
Can you leave a deprecation message if the user passes this in?

Reply:
Yes good point; done